config/quality/pmd/pmd-ruleset.xml

```diff
@@ -20,8 +20,8 @@
     <rule ref="rulesets/java/naming.xml">
         <!--<exclude name="AbstractNaming" />-->
         <exclude name="LongVariable" />
-        <!--<exclude name="ShortMethodName" />-->
-        <!--<exclude name="ShortVariable" />-->
+        <exclude name="ShortMethodName" />
```

Comment:
Yes, I agree `s` or `c` is not proper name, however how about `id`?

```
    public static void showServerConfigActivity(Context context, String id) {
```

With the ShortVariable rule, the code above would raise error. But I thought

```
    public static void showServerConfigActivity(Context context, String serverConfigId) {
```

is a little redundant.